scripts/refactor_testing_query_courseid.py

```diff
@@ -0,0 +1,176 @@
+import requests
```

Comment:
please add `#!/usr/bin/env python3` as the first line in the script, followed by a blank like.  this will allow us to run the script as `./whatever.py` instead of requiring `python3 whatever.py`